src/backend/src/main/kotlin/org/icpclive/overlay/Routing.kt

```diff
@@ -28,13 +29,52 @@ fun Route.configureOverlayRouting() {
     flowEndpoint("/mainScreen") { DataBus.mainScreenFlow.await() }
     flowEndpoint("/contestInfo") { DataBus.currentContestInfoFlow() }
     flowEndpoint("/runs") { DataBus.contestStateFlow.await().map { it.runsAfterEvent.values.sortedBy { it.time } } }
+    webSocket("/teamRuns") {
+        val teamIdStr = (incoming.receive() as? Frame.Text)?.readText()
```

Comment:
If we don't want to be able to subscribe to more teams over the existing webSocket, why send this instead of adding to name?